paramiko/transport.py

```diff
@@ -2065,6 +2065,11 @@ def run(self):
                         continue
                     if len(self._expected_packet) > 0:
                         if ptype not in self._expected_packet:
+                            if ptype == 30:
```

Comment:
there should probably be a name defined for this in `common.py`, just like `MSG_DISCONNECT`, `MSG_IGNORE`, `MSG_DEBUG` etc

Reply:
I have added MSG_KEXSKIP. I have to check, if 30 is the right value but I can't find information in the rfc.